cvelib/cli.py

```diff
@@ -541,7 +541,7 @@ def show_cve(ctx: click.Context, show_record: bool, print_raw: bool, cve_id: str
         # Serialize the CVE ID data and CVE record into an array for easier parsing if we're
         # showing the record as well. If not, just display the CVE ID data.
         if show_record:
-            print_json_data([cve_id_data, cve_record_data])
+            print_json_data(cve_record_data)
```

Comment:
@MrSeccubus Would you mind also updating the comment above to not mention the array approach any more? Perhaps something like:

```
        # Display CVE record data only if we're showing the record. Otherwise, show the CVE ID
        # data only.
```

Reply:
Done.